build.gradle

```diff
@@ -66,12 +66,17 @@ task javadocJar(type: Jar) {
     archiveClassifier = "javadoc"
 }
 
+ext {
+    // Load the Smithy Language Server version from VERSION.
+    libraryVersion = project.file("VERSION").getText('UTF-8').replace(System.lineSeparator(), "")
+}
+
```

Comment:
Can the script have [a `println`](https://github.com/awslabs/smithy/blob/main/build.gradle#L34) that says what version it's going to load?